.github/meta_issue_template/release-1.1.1-plugin-issues.md

```diff
@@ -0,0 +1,35 @@
+Coming from [opensearch-build#870](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+### You can find all the date in above issue 
+### (We only make changes to OpenSearch-Dashboards to 1.1.1, therefore, OpenSearch stays 1.1.0.)
+
+### Preparation
+
+- [ ] Assign this issue to a release owner.
+- [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).
+- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.
+
+### CI/CD
+
+- [ ] Increment version on main to `1.1.1.0`.
+- [ ] Ensure working and passing CI.
+- [ ] Re(add) this repo to the [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1).
+
+### Pre-Release
+
+- [ ] Branch and build from a `1.1` branch.
+- [ ] Update your branch in the [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1).
+- [ ] Feature complete, pencils down.
+- [ ] Apply sanity testing, and update results in the comment, contact corresponding assigner in meta issue above if needed.
```

Comment:
How about `Integration tests completed`, I do not like the term sanity testing, we should be making sure everything works not just kicking the tires.

The follow up comment that you added, should we put that as a note on top of the ticket instead of as part of this line item?

Reply:
Make changes, and will keep the current comment location as it is. Since they are logically apply in order.